pkg/chart/loader/load.go

```diff
@@ -102,7 +102,11 @@ func LoadFiles(files []*BufferedFile) (*chart.Chart, error) {
 
 		// Deprecated: requirements.yaml is deprecated use Chart.yaml.
 		// We will handle it for you because we are nice people
+		// Only supported for apiVersion v1
 		case f.Name == "requirements.yaml":
+			if c.Metadata.APIVersion != chart.APIVersionV1 {
+				return c, errors.Errorf("dependencies in requirements.yaml are not supported in apiVersion '%s'", c.Metadata.APIVersion)
```

Comment:
requirements.yaml is deprecated, but we will continue to support it in v2. We could probably inform the user that they should migrate to using Chart.yaml instead but we still should support this if they bump versions. See line 104 :)

Reply:
Good point @bacongobbler. Someone may bump it but maintain the requirements file. Updated accordingly.